src/crypto/CHIPCryptoPALPSA.cpp

```diff
@@ -43,6 +43,12 @@
 #include <string.h>
 #include <type_traits>
 
+#if CHIP_CRYPTO_PSA_AEAD_SINGLE_PART
+#define PSA_AEAD_MAX_PLAINTEXT CHIP_CONFIG_DEFAULT_UDP_MTU_SIZE
+#define PSA_AEAD_MAX_TAG 16
+#define PSA_AEAD_TEMP_BUFFER_SIZE (PSA_AEAD_MAX_PLAINTEXT + PSA_AEAD_MAX_TAG)
```

Comment:
nit: would c++ style `constexpr size_t kPsaAeadTempBufferSize = ...` for all these constants be more type safe mabye?

Reply:
I do agree, changes applied.